profiler/src/main/java/com/splunk/opentelemetry/profiler/ProfilingDataType.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.profiler;
+
+import java.util.Locale;
+
+public enum ProfilingDataType {
+  ALLOCATION,
+  PROFILING;
```

Comment:
`PROFILING` is not a nice name. Allocation is also profiling. Let's use `CPU` or `SAMPLER` instead.

Reply:
Backend is using `CPU` for this.